### PR TITLE
 Change diffdeb so as to ignore file timestamps.

### PR DESCRIPTION
**Pull request by schuyler**
This is necessary now that we're building packages in CI. CircleCI regularly makes a fresh checkout of the code base, with the result that file timestamps are different from one CI run to the next, because git doesn't set them.

This change ensures that file timestamps are never a reason in and of themselves to retain an otherwise unchanged Debian package.

This PR also adds a verbose flag to `diffdeb` for debugging purposes. This flag shouldn't be used by automated processes, since it leaves the temporary directories intact for manual investigation.

### CODE REVIEW

**Comment by schuyler** [comment]
Merging this as it is a straightforward change and its absence is causing havoc in our builds project.

File: tools/diffdeb [hunk 1]
```diff
@@ -10,20 +10,36 @@
...
+if [ "$verbose" == "-v" ]; then
+    shift
+else
+    verbose=""
+fi
 a=$1
 b=$2
 if [ "$b" = "" ]; then
-    echo "Usage: $0 <file1.deb> <file2.deb>"
+    echo "Usage: $0 [-v] <file1.deb> <file2.deb>"
```
Comment (zestyping) [inline tools/diffdeb]: Please update the help string ("identical in all aspects..." etc.)

File: tools/diffdeb [hunk 2]
```diff
@@ -37,16 +53,13 @@ function unpack_and_remove_version() {
     rm deb
 
     # Unzip data.tar to remove the gzip timestamp from the header
-    gunzip data.tar.gz
+    $TAR xfz data.tar.gz
```
Comment (zestyping) [inline tools/diffdeb]: Please update the comments on line 46–49 and line 55.

File: tools/diffdeb (hunk 1 above)
Comment (schuyler) [reply tools/diffdeb]: committed to dev as 54f456f1

File: tools/diffdeb (hunk 2 above)
Comment (schuyler) [reply tools/diffdeb]: committed to dev as 12a180a7 (oops)

**Pull request merged**